Fix warning during the build

```
    - WARN  | xcodebuild:  SwiftCentrifuge/Sources/SwiftCentrifuge/WebSocket/NativeWebSocket.swift:203:9: warning: switch covers known cases, but 'URLSessionTask.State' may have additional unknown values, possibly added in future versions
    - NOTE  | xcodebuild:  SwiftCentrifuge/Sources/SwiftCentrifuge/WebSocket/NativeWebSocket.swift:203:9: note: handle unknown values using "@unknown default"
    - NOTE  | [iOS] xcodebuild:  warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the App editor. (in target 'App' from project 'App')
```

--- a/Sources/SwiftCentrifuge/WebSocket/NativeWebSocket.swift
+++ b/Sources/SwiftCentrifuge/WebSocket/NativeWebSocket.swift
@@ -209,6 +209,8 @@ private extension URLSessionTask.State {
                 return "cancelling"
             case .completed:
                 return "completed"
+            @unknown default:
+                return "unknown"
         }
     }
 }

